include/deal.II/lac/la_vector.h

```diff
@@ -46,6 +46,20 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+/**
+ * A namespace for vector classes.
```

Comment:
Is it really only vectors? Did we ever discuss placing matrices and other classes in there as well?

Reply:
We don't have any new matrix classes (the new Epetra vectors work with the old matrices for example). I am not sure if this will change in the future though...

Reply:
The current list:
```
class  	CommunicationPatternBase
class  	ReadWriteVector
class  	Vector
class  	VectorSpaceVector
namespace CUDAWrappers:
    class  	Vector
    namespace kernel: (some structs)
namespace distributed:
    class  	BlockVector
    class  	Vector
namespace EpetraWrappers:
    class  	CommunicationPattern
    class  	Vector
```

Reply:
OK, let's not worry about it then.